receptorctl/receptorctl/cli.py

```diff
@@ -401,10 +409,10 @@ def cancel(ctx, unit_ids):
 @click.pass_context
 def release(ctx, force, unit_ids):
     if len(unit_ids) == 0:
-        print("No unit IDs supplied: Not doing anything")
+        printWarning("No unit IDs supplied: Not doing anything")
         return
     op = "release" if not force else "force-release"
-    print("Released:")
+    click.echo("Released:")
```

Comment:
I like that you added helper functions for errors and warnings, but dislike that we `click.echo` repeated everywhere. I don't care what you call the new function (`print_info`?), but we should apply a similar pattern here to future-proof this code a bit.

Reply:
Initially I only added error and warning to make sure we got a consistent color and prefix. But added a generic print_message function as well.